packages/hierarchies-react/src/presentation-hierarchies-react/internal/Utils.ts

```diff
@@ -63,3 +64,23 @@ export function safeDispose(disposable: {} | { [Symbol.dispose]: () => void } |
     disposable[Symbol.dispose]();
   }
 }
+
+/** @internal */
+export function debounceWrapper<T, R>(pipeline: (value: T) => Observable<R>, next: (value: R) => void, complete: () => void): Subject<T> {
+  const subject = new Subject<T>();
+  subject
+    .pipe(
+      debounceTime(50),
```

Comment:
Would changing to `0` here stop it from working?

Reply:
It seems to work (no double reload at initial load, tests pass). My only problem with it would be that if there are enough reload calls this would fail and reload multiple time as there is some code before this, but I guess with smaller wait debounce time it should improve overall performance, changed to 0. PS. could not find if the 0 defaults to 1ms or something else.